src/components/CropSelector/CropSidebar.js

```diff
@@ -121,25 +132,14 @@ const CropSidebarComponent = (props) => {
     setShowFilters(value);
   }, [debug, state.speciesSelectorActivationFlag, from, comparisonView]);
 
-  const [cashCropVisible, setCashCropVisible] = useState(true); // TODO: buggy(?)
-
-  const [dateRange, setDateRange] = useState({
-    startDate: null,
-    endDate: null,
-  });
-
-  const [sidebarFilters, setSidebarFilters] = useState([]);
-
-  const dateRangeModalOpen = () => {
-    change('TRUE', null, 'dateRangeOpen');
-  };
-
-  // make an exhaustive array of all params in array e.g. cover crop group and use includes in linq
-  const [sidebarFilterOptions, setSidebarFilterOptions] = useState(() => {
-    const sidebarStarter = {};
-    sidebarFiltersData.forEach((row) => (sidebarStarter[row.name] = []));
-    return sidebarStarter;
-  });
+  const handleToggle = (value, type = 'TOGGLE') => {
```

Comment:
should this be named something like dispatchWrapper since it's used for the modal open as well?